cmake/iwyu.cmake

```diff
@@ -20,15 +24,56 @@ if(NOT WIN32)
     # That's the public facing include-what-you-use target which can be invoked by `make iwyu`.
     # It generates the include-what-you-use report and writes it into iwyu_results.txt in the
```

Comment:
I guess the comment needs to be updated too. We are not creating `iwyu_results.txt` anymore, aren't we?

Reply:
Yessss... Thanks. Will fix that!

Reply:
Done.